principles.html

```diff
@@ -308,18 +318,27 @@ <h4 class="f_panelHead">Accessibility requirements related to adaptability <br /
     <ul class="listwithp">
       <li><a href="http://www.w3.org/WAI/WCAG20/quickref/#content-structure-separation" shape="rect">Guideline 1.3 - Adaptable</a></li>
     </ul>
-    <p class="listintro"><strong><acronym>UAAG</acronym> 1.0</strong></p>
+    <p class="listintro"><strong><acronym>UAAG</acronym> 2.0</strong></p>
     <ul class="listwithp">
-      <li><a href="http://www.w3.org/TR/UAAG10/guidelines.html#gl-user-control-styles" shape="rect">Guideline 4 - Ensure user control of rendering</a></li>
-      <li><a href="http://www.w3.org/TR/UAAG10/guidelines.html#gl-user-control-ui" shape="rect">Guideline 5 - Ensure user control of user interface behavior</a></li>
-      <li><a href="http://www.w3.org/TR/UAAG10/guidelines.html#gl-configuration" shape="rect">Guideline 11 - Allow configuration and customization</a></li>
+      <li>original:<a href="http://www.w3.org/TR/UAAG10/guidelines.html#gl-user-control-styles" shape="rect"><del>Guideline 4 - Ensure user control of rendering</del></a></li>
+       <li>original:<a href="http://www.w3.org/TR/UAAG10/guidelines.html#gl-user-control-ui" shape="rect"><del>Guideline 5 - Ensure user control of user interface behavior</del></a></li>
+      <li>original:<a href="http://www.w3.org/TR/UAAG10/guidelines.html#gl-configuration" shape="rect"><del>Guideline 11 - Allow configuration and customization</del></a></li>
+      <li>replaced 4, 5 &amp; 11 with 1.4 - 1.7: <a href="https://www.w3.org/TR/UAAG20/#gl-text-config" shape="rect">Guideline 1.4 - Provide text configuration</a></li>
+      <li>replaced 4, 5 &amp; 11 with: <a href="https://www.w3.org/TR/UAAG20/#gl-volume-config" shape="rect">Guideline 1.5 - Provide volume configuration</a></li>
+      <li>replaced 4, 5 &amp; 11 with: <a href="https://www.w3.org/TR/UAAG20/#gl-speech-config" shape="rect">Guideline 1.6 - Provide synthesized speech configuration</a></li>
+      <li>replaced 4, 5 &amp; 11 with: <a href="https://www.w3.org/TR/UAAG20/#gl-style-sheets-config" shape="rect">Guideline 1.7 - Enable configuration of user stylesheets</a></li>
+     
```

Comment:
I did not check the content but this seems like exactly the right type of mapping! Please use this approach throughout, but replace the bullets rather than keep "original" and "replaced" for each change.